www/js/router.js

```diff
@@ -1,275 +1,276 @@
-      templateUrl: 'templates/profile/profile.html',
-      controller: 'profile'
-    }, {
-      name: 'profile2',
-      url: '/profile-2',
-      templateUrl: 'templates/profile/profile-2.html',
-      controller: 'profile-step2'
-    }, {
-      name: 'profile3',
-      url: '/profile-3',
-      templateUrl: 'templates/profile/profile-3.html',
-      controller: 'profile-step3'
-    }, {
-      name: 'settings',
-      url: '/settings',
-      templateUrl: 'templates/profile/settings.html',
-      controller: 'settings'
-    }, {
-      name: 'messages',
-      url: '/messages',
-      templateUrl: 'templates/messages/list.html',
-      controller: 'messages'
-    }, {
-      name: 'microPetionsAdd',
-      url: '/micro-petitions/add/:type/',
-      templateUrl: 'templates/petitions/add.html',
-      controller: 'petitions.add'
-    }, {
-      name: 'microPetionsGroupAdd',
-      url: '/micro-petitions/add/:type/:group_id',
-      templateUrl: 'templates/petitions/add.html',
-      controller: 'petitions.add'
-    }, {
-      name: 'groupPetions',
-      url: '/group-petitions',
-      templateUrl: 'templates/home/home.html',
-      controller: 'home'
-    }, {
-      name: 'groupPetion',
-      url: '/group-petions/:id',
-      templateUrl: 'templates/petitions/group.html',
-      controller: 'petitions.group'
-    }];
-
-  states.forEach(function (state) {
-    if (!state.name) {
-      return;
-    }
-    var options = {cache: !!state.cache, url: state.url};
-    options.views = {
-      'menuContent': {
-        templateUrl: state.templateUrl
-      }
-    };
-    if (state.controller) {
-      options.views.menuContent.controller = state.controller;
-    }
-    $stateProvider.state('app.' + state.name, options);
-  });
-
-  $urlRouterProvider.otherwise('/preload');
-
-});
+angular.module('app')
+.config(function ($stateProvider, $urlRouterProvider/*, $routeProvider*/) {
+  $stateProvider
+    .state('app', {
+      url: '',
+      abstract: true,
+      templateUrl: 'templates/menu.html',
+      controller: 'AppCtrl'
+          });
+
+  var states = [{
+      name: 'preload',
+      url: '/preload',
+      templateUrl: 'templates/home/preload.html',
+      controller: 'preload'
+    }, {
+      name: 'main',
+      url: '/main',
+      cache: true,
+      templateUrl: 'templates/home/home.html',
+      controller: 'home'
+    }, {
+      name: 'newActivities',
+      url: '/new-activities',
+      cache: true,
+      templateUrl: 'templates/home/home.html',
+      controller: 'home'
+    }, {
+      name: 'login',
+      url: '/login',
+      templateUrl: 'templates/session/login.html',
+      controller: 'session.login'
+    }, {
+      name: 'logout',
+      url: '/logout',
+      templateUrl: 'templates/session/logout.html',
+      controller: 'session.logout'
+    }, {
+      name: 'commingSoon',
+      url: '/comming-soon',
+      cache: true,
+      templateUrl: 'templates/coming-soon.html'
+    }, {
+      name: 'poling',

```

Comment:
ok thanks. we may need to swing back to this one another time. if its a typo that is now functioning, we'll just keep with what had previously been there.